taf/updater/updater.py

```diff
@@ -643,14 +635,13 @@ def _commits_ret(commits, existing_repo, update_successful):
             if auth_repo_name is not None
             else True
         )
-
         # first clone the validation repository in temp. this is needed because tuf expects auth_repo_name to be valid (not None)
         # and in the right format (seperated by '/'). this approach covers a case where we don't know authentication repo path upfront.
         auth_repo_name = _clone_validation_repo(url, auth_repo_name, default_branch)
-
-        repository_updater = tuf_updater.Updater(
-            auth_repo_name, repository_mirrors, GitUpdater
-        )
+        git_updater = GitUpdater(url, clients_auth_library_dir, auth_repo_name)
+        _run_tuf_updater(git_updater)
+    except UpdateFailedError as e:
```

Comment:
Not sure why this particular error is handled differently? Don't we want to return `Event.FAILED` and other data?

Reply:
My bad, we do want to return `Event.FAILED`. Removed that specific except